src/dataset-builder/ConceptSetCreator.test.ts

```diff
@@ -7,6 +7,8 @@ import { ConceptSetCreator, toConceptSet } from './ConceptSetCreator';
 import { homepageState } from './dataset-builder-types';
 import { dummyDatasetModel } from './TestConstants';
 
+jest.mock('src/libs/ajax/GoogleStorage');
```

Comment:
This change seems unrelated, should this have been here before? Or am I missing something about how it connects to the refactor?

Reply:
I think it should've been there before. This is required for the initialization of the tests, I'm not sure exactly how they were working before but it might have been fragile because after the refactor, this is needed to run the tests.